### PR TITLE
Prepare version 4.56

### PR DESCRIPTION
**Pull request by northwestwitch**
### Added
- [x] Test for PanelApp panels loading
- [x] `panel-umi` tag option when loading cancer analyses
### Changed
- [x] Black text to make comments more visible in dark mode
- [x] Loading PanelApp panels replaces pre-existing panels with same version
- [x] Removed sidebar from Causatives page - navigation is available on the top bar for now
### Fixed
- [x] Remove a:visited css style from all buttons
- [x] Update of HPO terms via command line
- [x] Background color of MIXED and PANEL-UMI sequencing types on cases page
- [x] Fixed regex error when searching for cases with query ending with `\ `

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR, DN

### CODE REVIEW

**Comment by northwestwitch** [comment]
I'll install on stage, so we test it before releasing!

**Comment by northwestwitch** [comment]
We could also install ot on hasta stage so we can test updating all the collections from there.

**Comment by northwestwitch** [comment]
Ok, the update of scout collection doesn't crash or show problems on hasta stage. The numbers of items saved in the collections seems legit as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Update of HPO terms via command line
 - Background color of `MIXED` and `PANEL-UMI` sequencing types on cases page
 - Fixed regex error when searching for cases with query ending with `\ `
-- Genesymbols on Causatives page lighter in dark mode
+- Gene symbols on Causatives page lighter in dark mode
```
Comment (dnil) [inline CHANGELOG.md:20]: 👍

**Review by dnil — APPROVED** [review]

**Comment by dnil** [comment]
And deployed ok on `cg-prod-services`:
```
[hiseq.clinical@cg-prod-services ~]$ systemctl --user start scout@v4.56
```

**Pull request merged**